.github/workflows/merge-main-into-staging.yml

```diff
@@ -0,0 +1,21 @@
+name: Merge main into staging
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  merge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Merge main into staging
```

Comment:
Might rename this to indicate the underlying intention: "Trigger release candidate build and deploy". In fact maybe rename the entire workflow similarly?

Reply:
will fix